### ISSUE TITLE
Dome texture overlaps city names

### ISSUE THREAD

**Issue by Cervator** [Terasology/LightAndShadow#58]
Noticed by chance during unrelated testing - the texture used by the magic dome in Light & Shadow seems to draw in front of a floating city name.

That might be more of a z-order engine sorting/support issue, but I noticed it here and am not sure. Just don't want to forget - low priority tho :-)

![2017-07-27 01_53_16-terasology _ alpha](https://user-images.githubusercontent.com/1063833/28656174-6117239a-726e-11e7-81dd-96d7ae0d4211.png)

**Comment by jdrueckert** [comment]
@Cervator I guess if we still had cities in LaS, this issue would still persist... As we don't, should we consider closing it?

**Comment by Cervator** [comment]
It isn't LaS related, no, so this can indeed be closed. We should check to verify it and if still valid open an engine issue instead (with whatever replication steps - maybe that could even be Hacktoberfesty to fix somehow ... probably not GFI since I'm not sure how to directed somebody overly new)

**Comment by jdrueckert** [comment]
@Cervator If it is an engine / rendering issue (which is likely), then yes. But for that we would probably need to find a way to reproduce this. If it indeed is a z-order sorting/support issue, we should be able to see this in other settings as well, e.g. if two cities are close to one another and we use `ghost` to align their names, the name of the nearest one should be overlapped by the name of the one farther away...
Also, this might be related to a bug I experienced with the `WorldlyTooltip` in MR:
If you look at a gooey and there's only air behind it (e.g. frog perspective or if it's somewhere above you), then the tooltip shows the gooey's name. If you look at a gooey and there's some block behind it, you'll see the name/type of the block instead. I always thought this would be a bug in `WorldlyTooltip` but if we indeed had a z-order sorting issue, that _could_ also cause this, right?
